cmd/soroban-rpc/internal/preflight/preflight_test.go

```diff
@@ -362,18 +363,23 @@ func benchmark(b *testing.B, config benchmarkConfig) {
 			disableCache: config.useDB.disableCache,
 		}
 	}
+	var fullDuration time.Duration
+	b.ResetTimer()
 	b.StopTimer()
 	for i := 0; i < b.N; i++ {
 		params := getPreflightParameters(b, dbConfig)
+		start := time.Now()
 		b.StartTimer()
 		_, err := GetPreflight(context.Background(), params)
 		b.StopTimer()
+		fullDuration += time.Since(start)
```

Comment:
The time metric reported by the Go benchmark package is based on the same calculation, a `time.Since` between the `time.Now` in `b.StartTimer` and `b.StopTimer`. For example: https://cs.opensource.google/go/go/+/refs/tags/go1.21.0:src/testing/benchmark.go;l=127-137. Is the metric being added in this PR different in some other way?

Reply:
Thanks that's why I was noting very similar results. I was assuming it was CPU time.

I removed that part of the PR.